src/openapi/streaming/streaming.ts

```diff
@@ -738,6 +740,35 @@ class Streaming extends MicroEmitter<EmittedEvents> {
         this.orphanFinder.update();
     }
 
+    /**
+     * If we get 3 resets within 1 minute then we wait for 1 minute
+     * since it may indicate some problem with publishers
+     */
+    private checkIfPublishersDown(resetTimeStamp) {

```

Comment:
This is across all subscriptions- I would rather this were per subscription. It might make some of the subscription logic easier too as you can directly put it into a new state.

Reply:
changed to per subscription as suggested